pinot-broker/src/test/java/org/apache/pinot/broker/querylog/QueryLoggerTest.java

```diff
@@ -259,11 +259,9 @@ private QueryLogger.QueryLogParams generateParams(boolean isGroupLimitHit, int n
     response.setExceptions(
         IntStream.range(0, numExceptions)
             .mapToObj(i -> new ProcessingException()).collect(Collectors.toList()));
-    response.setOfflineTotalCpuTimeNs(14);
```

Comment:
Are these 2 removal intentional?

Reply:
Yes, I've removed these `setOflineTotalCpuTimeNs` and `setRealtimeTotalCpuTimeNs` from `BrokerResponseNative`. In production code they are always set as `threadCpuTimeNs + systemActivitiesCpuTimeNs + responseSerializationCpuTimeNs`, so it looks error prone to have a setter (like the one we used here). Instead we never set the value of the total and always derive it from its parts.

Reply:
BTW, this is why in lines 110 an 111 I had to modify the result. Before we were returning the synthetic values 14 and 18 and now we return 48 and 60 which is the sum of 15, 16 and 17 and 19, 20 and 21 respectively.